query/src/main/scala/filodb/query/exec/ExecPlan.scala

```diff
@@ -188,13 +188,14 @@ trait ExecPlan extends QueryCommand {
       rv : Observable[RangeVector], recordSchema: RecordSchema, resultSchema: ResultSchema
     ): Task[QueryResult] = {
         @volatile var numResultSamples = 0 // BEWARE - do not modify concurrently!!
+        @volatile var resultSize = 0L
```

Comment:
Unsure why `volatile` was needed above (afaik `map` is single-threaded), so I'm mirroring it here just-in-case.